forge-gui/res/cardsfolder/upcoming/sharp_eyed_rookie.txt

```diff
@@ -0,0 +1,13 @@
+Name:Sharp-Eyed Rookie
+ManaCost:1 G
+Types:Creature Human Detective
+PT:2/2
+K:Vigilance
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.YouCtrl+powerGTX,Creature.YouCtrl+powerGTY | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever a creature enters the battlefield under your control, if its power is greater than CARDNAME's power or its toughness, put a +1/+1 counter on CARDNAME and investigate. (Create a Clue token. It's an artifact with "{2}, Sacrifice this artifact: Draw a card.")
```

Comment:
I'm going to need more detail on what's wrong with the Valid. (Important note: Sharp-Eyed Rookie only cares about the power of the creature entering, which is a major thing separating it from Evolve.)

Reply:
hmn right, I misread it to compare both toughness together too

Reply:
actually it turns out your Oracle is incomplete

Reply:
@tool4ever What a careless mistake! I must have somehow glanced over a whole line of text.